tests/30rooms/06membership.pl

```diff
@@ -1,4 +1,5 @@
 use List::Util qw( first );
+use Digest::SHA qw( sha256_hex );
```

Comment:
As this is now a new dependency, can you add an appropriate line into the `cpanfile`?

    requires "Digest::SHA";